daml/Marketplace/Distribution/Bid/Model.daml

```diff
@@ -0,0 +1,45 @@
+module Marketplace.Distribution.Bid.Model where
```

Comment:
Maybe rename the directory to `Bidding` instead of `Bid` - BiddingService sounds better imo.

Reply:
Done